File: csharp/bypstest-ser/src-ser/com/wilutions/byps/test/api/inherit/FruitService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using com.wilutions.byps;

namespace com.wilutions.byps.test.api.inherit
{
	
	// checkpoint com.wilutions.byps.gen.cs.GenRemoteClass:93
	public interface FruitService : BRemote, 
		PlantService {
	
		String Squeeze();
		void Squeeze(BAsyncResult<String> asyncResult) ;
		Task<String> SqueezeAsync();
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/inherit/FruitService.java
package byps.test.api.inherit;

import byps.RemoteException;

/**
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 *
 *
 */
public interface FruitService extends PlantService {

  public String squeeze() throws RemoteException;
  
  
}
